source/octf/interface/InterfaceTraceManagementImpl.cpp

```diff
@@ -51,6 +51,8 @@ void InterfaceTraceManagementImpl::listTraces(
 
     // Check for trace summaries in these directories
     proto::TraceSummary summary;
+    traceDirs.sort();
```

Comment:
Later (after submodule sync) we can also removing sorting in iotracer tests since it will no longer be needed.